Add blog posts for HODL

at2005 opened the pull request.

willzeng commented:
Hi @at2005 Thanks for posting! Is this ready for review?

nathanshammah reviewed (changes requested):
Thanks @at2005! Excellent job. Some minor technicalities:
- Can you please generate the html page, using the pandoc commands reported in the [readme](https://github.com/unitaryfund/unitary-fund#writing-a-blog-post).
Please add a link to the html page to the blog index, blog.html?
- Also, please have a double check at the html file locally to check that pandoc converted it correctly, e.g., https://github.com/unitaryfund/unitary-fund/issues/243.
- What md file needs to be included, there are two now
- there is an image at the top that I think is not supposed to be there?
- I'm not familiar with the way images are included: can they included in the website for more reliability in the future.

at2005 commented:
Hi, I've updated the files accordingly :) There is now one HTML file (hodl.html), and have edited it to format it properly and added the link to blog.html.

nathanshammah commented on posts/_markdown/qiskit_hodl.md:
```diff
@@ -0,0 +1,105 @@
+
+](https://medium.com/@atamb?source=post_page-----377859287861--------------------------------)[Ayush Tambde](https://medium.com/@atamb?source=post_page-----377859287861--------------------------------)Follow
+
+Jun 28
```
```suggestion
2022-08-19
```

nathanshammah commented on posts/hodl.html:
```diff
@@ -0,0 +1,333 @@
+                                with HODL</h3>
+                            </right>
+                            <div>
+                                <p>Jun 28</p>
```
```suggestion
                                <p>2022-08-19</p>
```

nathanshammah commented on blog.html:
```diff
@@ -71,6 +71,27 @@
               <section class="light">
                 <ul style="list-style-type: none">
+                  <li class="leading-block">
+                    <span class="blog-post-date">7 August, 2022</span>
```
```suggestion
                    <span class="blog-post-date">19 August, 2022</span>
```
